frontend/public/components/monitoring/targets.tsx

```diff
@@ -293,9 +320,34 @@ const REDUX_ID = 'monitoringTargets';
 
 const getRowProps = (target: Target) => ({ id: target.scrapeUrl, title: target.lastError });
 
+const getWatchError = (monitorsWatchContext, t) => {
+  const loadErrorIndex = 2;
+  const watchError = monitorsWatchContext[loadErrorIndex];
+  const authInfo =
+    'https://docs.openshift.com/container-platform/4.11/monitoring/enabling-monitoring-for-user-defined-projects.html';
```

Comment:
We will need to do something other than hardcoding the OpenShift version in the URL (`4.11`) so that we don't need to update this every version.

Reply:
Replacing `4.11` with `latest` allows rerouting to the latest docs version of OCP. I'll update this in the next PR.